admin-dev/themes/new-theme/scss/pages/product/product_page.scss

```diff
@@ -1075,6 +1075,40 @@ $product-page-padding-bottom: 80px !default;
       max-width: 415px;
     }
   }
+
+  #product_description_related_products {
+    .search-with-icon {
+      max-width: 377px;
+    }
+
+    .entities-list-container {
+      display: flex;
+      flex-direction: row;
+      flex-wrap: wrap;
+      margin: 0.5rem -0.5rem;
+    }
+
+    .related-product {
+      display: flex;
+      flex-direction: column;
+      width: 180px;
```

Comment:
Looks like a there is a problem there, we almost never use a fixed width anywhere because on responsive  it will be too small

Usually it's

```
width: 100%;
max-width: 180px;
```